docs/modules/ROOT/pages/includes/quarkus-artemis-core.adoc

```diff
@@ -10,38 +10,22 @@ h|[[quarkus-artemis-core_configuration]]link:#quarkus-artemis-core_configuration
 h|Type
 h|Default
 
-a|icon:lock[title=Fixed at build time] [[quarkus-artemis-core_quarkus.artemis.health.enabled]]`link:#quarkus-artemis-core_quarkus.artemis.health.enabled[quarkus.artemis.health.enabled]`
```

Comment:
Is it necessary to change config names? I thought we made it the same as other extensions, and it is a breaking change.

Reply:
I am still playing around with those options. Right now, it is challenging to determine at runtime which configurations exist since we do not have a configuration that must be present and is built-time fixated (like `db-kind` in datasources, for example).

Reply:
FTR: https://github.com/quarkusio/quarkus/issues/28430

Reply:
I aligned the config properties with the properties used by `quarkus.datasource`, i.e.:
- `quarkus.artemis.health.enabled` to enable health checking for artemis
- `quarkus.artemis.health-exclude` and `quarkus.artemis."named-connection".health-exclude` to exclude specific connections from health checking
- `quarkus.artemis.xa-enabled` and `quarkus.artemis."named-connection".xa-enabled` to activate XA transactions (beign in line with `quarkus.artemis.health-exclude`).

Sub-properties with dots in names, as described in the bug linked above, don't currently work. We can work around this by creating an object for `xa`, having a field `enable`. This would, however, be not in line with how `quarkus.datasource` does things.

Reply:
Thanks for the clarification. Alignment with other extensions, especially Quarkus' is always good (my preference is also dash over camelCase). I guess, it is easily fixable on upgrade, we just need to do a minor version bump.